bin/jaotc

```diff
@@ -0,0 +1 @@
+.openjdk@11.pkg
```

Comment:
are these bin files intentional?

Reply:
Yes, they're part of the Hermit config and meant to be checked in. For example in Misk: https://github.com/cashapp/misk/tree/master/bin

Reply:
ok it's not obvious at a glance I guess, this works better than a file listing deps?

Reply:
Yes, I initially was hesitant but over the past year it's made toolchain management a lot simpler across internal and external repos.